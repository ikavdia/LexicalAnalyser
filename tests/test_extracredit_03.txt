1Ax0
1x08
1234x08