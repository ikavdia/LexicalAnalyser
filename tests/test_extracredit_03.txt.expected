{1 , NUM , 1}
{Ax0 , ID , 1}
{1x08 , BASE08NUM , 2}
{1234x08 , BASE08NUM , 3}
{ , END_OF_FILE , 3}